### ISSUE TITLE
Remove react no-script

### ISSUE THREAD

**Issue by grzanka** [yaptide/ui#1412]
![image](https://github.com/yaptide/ui/assets/7374733/c5d02bd3-6588-408a-8867-6f5d3466d5d8)

Some technologies see react no-script, this should be removed
